content/References/IPv6.md

```diff
@@ -0,0 +1,247 @@
+
+  Flow labeling is a 20-bit part in the IPv6 header. It is intended to identify packets belonging to the same flow or traffic stream.
+  Flow labeling provides routers and network devices with a way to recognize and classify packets that are part of the same flow, such as packets that belong to a specific real time audio or video stream.
+  This allows for more efficient handling of packets within the network.
+
+  Despite potential benefits, flow labeling adoption is currently limited.
+  There is a lack of standardized QoS mechanisms and policies on how to make use of this field.
+  Flow labeling depends on widespread support and consistent implementation across network devices and protocols but it not implemented this way at present.
+{{< /expand >}}
+* {{< expand "Built-in encryption, authentication, and privacy support" "v" >}}
+
+  IPv6 includes built-in support for authentication and privacy features to enhance security and protect network. Two key mechanism for authentication and privacy are Secure Neighbor Discovery (SEND) and Internet Protocol Security (IPsec).
+
+  Secure Neighbor Discovery (SEND) is an extension of NDP.
+  It introduces cryptographic mechanisms for neighbor and routing authentication to ensure devices can authenticate the identity of neighboring devices and routers before establishing communication. This provides protection against various attacks such as neighbor spoofing and address theft.
+  It also includes mechanisms for securing IPv6 address autoconfiguration to prevent unauthorized devices from obtaining valid IPv6 addresses on the network.
+
+  Internet Protocol Security (IPsec) is a suite of protocols used to secure IPv6 communications at the IP layer.
+  IPsec originally defined two protocols for securing IP packets, Authentication Header (AH) and Encapsulating Security Payload (ESP). 
+  AH provides data integrity and anti-replay services
+  ESP provides encryption and authenticates data.
+  It can also includes Internet Key Exchange (IKE), which is used to generate shared security keys that establish security associations (SAs).
+  A special router or firewall sitting between two networks usually handles the security associations needed for the encryption/decryption process to negotiate a security level between the two entities.
+
+  It can establish secure tunnels between IPv6 hosts or networks, encrypting all traffic passing through the tunnel and providing protection against eavesdropping and tampering.
+  If implemented in transport mode, only the payload of IP packets is encrypted.
+  If implemented in tunnel mode, the entire IP packet including the original IP header is encapsulated and encrypted.
+  IPsec is widely supported in IPv6-capable devices, and is commonly used to secure communication over IPv6 networks, particularly enterprise and VPN environments.
+{{< /expand >}}
+* Easier implementation of services like peer-to-peer (P2P) networks and voice over IP
+
```

Comment:
Probably need to turn this into an expand if we remove the bullets from the others. Might be good just from a visual consistency standpoint to make it match the others anyway. Is there a short but of clarification you can add about how IPv6 simplifiest these installations?

Reply:
Made the suggested change, adding expands for each of the downside points. I think you mean adding more meet on the bone about peer to peer and Voice over IP, so I'm adding a bit of an explanation for these.